### PR TITLE
Added nick property to dispatcher args

### PR DESCRIPTION
**Pull request by CExMNemr**
Something like this @nherment ?

### CODE REVIEW

File: lib/ServiceRouter.js [hunk 1]
```diff
@@ -158,21 +158,23 @@ ServiceRouter.prototype._buildDispatcher = function() {
 
     var clientToken = req.seneca.cookies.get(self._options.tokenkey)
     var serverToken
+    var nick 
     res.seneca.cookies.set(self._options.tokenkey)
 
     if( req.seneca ) {
       serverToken = req.seneca.login && req.seneca.login.id
+      userNick = req.seneca.login && req.seneca.login.nick
```
Comment (nherment) [inline lib/ServiceRouter.js]: @CExMNemr ``userNick`` does not exist.

**Comment by nherment** [comment]
yes that's it. but the variable name is messed up. I can't merge as is.

File: lib/ServiceRouter.js (hunk 1 above)
Comment (CExMNemr) [inline lib/ServiceRouter.js]: ah! Yes I just saw it, thanks.

**Comment by nherment** [comment]
@CExMNemr published seneca-auth@0.2.16

**Pull request merged**